pkg/aws/iam.go

```diff
@@ -190,3 +196,27 @@ func unMarshallPolicyDocument(document string) (*PolicyDocument, error) {
 	}
 	return &policyDocument, nil
 }
+
+// getIAMRole returns the role object struct that has attributes of a given role
+func getIAMRole(ctx context.Context, client IAMClient, roleName string) (output *iam.GetRoleOutput, err error) {
+	input := &iam.GetRoleInput{
+		RoleName: &roleName,
+	}
+
+	output, err = client.GetRole(ctx, input)
+	if err != nil {
+		return output, err
+	}
+	log.Println(output.Role)
+
+	return output, err
+}
+
+// Asserts the MaxSessionDuration attribute of a given IAM Role
+func AssertIamRoleMaxSessionDuration(t *testing.T, ctx context.Context, client IAMClient, roleName string, maxDuration int32) {
+	output, err := getIAMRole(ctx, client, roleName)
+	assert.Nil(t, err)
+
+	assert.Equal(t, *output.Role.MaxSessionDuration, maxDuration)
```

Comment:
**Suggestion:** I would add a message to this `assert` statement, like `Role <rolename> has an incorrect max duration value.`. That way, if the test fails it is obvious which role and why.